### PR TITLE
Docs 8063 metrics archive api docs

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

File: modules/ROOT/pages/metrics-archive-api.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
+= Anypoint Monitoring Archive API
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Anypoint Monitoring Archive API enables you to download metric data for APIs. The data that this API archives appears in your xref:monitoring::api-analytics-dashboard.adoc#enhanced-api-metrics[enhanced API metrics dashboards]. However, the data contained in the archive is asynchronous.
+
+You must have a Titanium subscription to use this API.
```
Comment (jiang-wu) [inline modules/ROOT/pages/metrics-archive-api.adoc:8]: There is a rate limit on using this API. The rate limit is 18,000 calls per hour.

File: modules/ROOT/pages/metrics-archive-api.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+== How to Use Data
+
+Downloading metrics data using this API enables you to analyze the data or integrate it into your own system. You can write an integration application and use the API to navigate the data and discover files therein. The integration application can use the API's navigation structure to retrieve the data you need.
+
+The data archive has API metric JSON files that are organized in the following hierarchy:
+
+`/organizations/<organization>/environments/<environment>/<entityType>/<entityID>/<fileType>/<year>/<month>/<dayOfMonth>/<fileName>`
+
+For example, you can write your integration application such that it downloads information for a specific API using a path such as `/organizations/<org-id>/environments/<env-id>/apis/<api-id>`.
+
```
Comment (jiang-wu) [inline modules/ROOT/pages/metrics-archive-api.adoc:29]: Can say something how to make use of the date time part of the organization to download only data in specific date time range.  Or remember the last downloaded data time and only download any incrementally appearing new files.

Comment (jiang-wu) [reply modules/ROOT/pages/metrics-archive-api.adoc:29]: Maybe one more example on the reason behind file types.  Data are stored in different files by file type.  This way, the caller may choose to download only a subset of the data such as just getting the summary information.  By downloading a subset as needed, it makes the data processing more efficient for the caller.

File: modules/ROOT/pages/metrics-archive-api.adoc [hunk 3]
```diff
@@ -0,0 +1,33 @@
+= Anypoint Monitoring Archive API
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Anypoint Monitoring Archive API enables you to download metric data for APIs. The data that this API archives appears in your xref:monitoring::api-analytics-dashboard.adoc#enhanced-api-metrics[enhanced API metrics dashboards]. However, the data contained in the archive is asynchronous.
```
Comment (jiang-wu) [inline modules/ROOT/pages/metrics-archive-api.adoc:6]: Should we document the supported Mule runtime versions?  Not all versions are supported.

File: modules/ROOT/pages/metrics-archive-api.adoc [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+You must have a Titanium subscription to use this API.
+
+== File Structure and Downloads
+
+The Anypoint Monitoring Archive API uses a directory hierarchy. At the lowest level, you can access and download files containing archived data from your enhanced API metrics dashboards.
```
Comment (jiang-wu) [inline modules/ROOT/pages/metrics-archive-api.adoc:12]: We can also point out that the user should be familiar with enhanced API Metrics, the supported Mule runtime versions, and have used advanced custom dashboard features before proceeding with Archive API. By using those, the user will then have the right understanding of the content provided from Archive API as you can see the same content through advanced custom dashboard.

File: modules/ROOT/pages/api-analytics-dashboard.adoc [hunk 5]
```diff
@@ -26,9 +26,15 @@ image::api-analytics-detail-panel.png[]
 +
 . If you want to view API analytics in API Manager, click the *View API in API Manager* link.
 
+[[enhanced-api-metrics]]
 == Enhanced Metrics View for API Analytics Dashboards
```
Comment (myuin-ms) [inline modules/ROOT/pages/api-analytics-dashboard.adoc:30]: I don't understand this title and how it's related to the archive API.

File: modules/ROOT/pages/api-analytics-dashboard.adoc [hunk 6]
```diff
@@ -26,9 +26,15 @@ image::api-analytics-detail-panel.png[]
...
 
-API analytics dashboards include various pages to view different types of charts:
+API analytics dashboards include various pages to view different types of charts.
+
+Titanium users who use the enhanced metrics view for API analytics can use the Anypoint Monitoring Archive API to download metric data that appears in the charts.
```
Comment (myuin-ms) [inline modules/ROOT/pages/api-analytics-dashboard.adoc:34]: How about this:
Titanium users who use the enhanced API analytics dashboards can also access the data directly via the Anypoint Monitoring Archive API. With this powerful API, you can navigate and discover your API metrics data using a directory hierarchy. The API allows download of files that contains JSON objects which represent........

File: modules/ROOT/pages/api-analytics-dashboard.adoc (hunk 5 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/api-analytics-dashboard.adoc:30]: The heading is for enhanced metrics view and not specifically the API. The basic view is also described in this topic. So, in short, it isn't.

File: modules/ROOT/pages/api-analytics-dashboard.adoc (hunk 6 above)
Comment (sarathecoubian) [inline modules/ROOT/pages/api-analytics-dashboard.adoc:34]:
```suggestion
Titanium users who use the enhanced API analytics dashboards also access the data directly via the Anypoint Monitoring Archive API. With this API, you can navigate and discover your API metrics data using a directory hierarchy.
```

File: modules/ROOT/pages/api-analytics-dashboard.adoc [hunk 7]
```diff
@@ -26,9 +26,15 @@ image::api-analytics-detail-panel.png[]
...
+Using a directory hierarchy, the API allows access to text files that contains https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-monitoring-archive-api/minor/1.0/pages/Metric%20Data/[JSON objects] that represent one or more metric events. You can also write an application to navigate the data collected by the API and integrate it into your own system. For more information about this API, see the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-monitoring-archive-api/minor/1.0/pages/home/[Anypoint Monitoring Archive API] page in Anypoint Exchange.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/api-analytics-dashboard.adoc:35]:
```suggestion
The API allows you to download files that contain https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-monitoring-archive-api/minor/1.0/pages/Metric%20Data/[JSON objects] that represent one or more metric events. You can also write an application to navigate the data collected by the API and integrate it into your own system. For more information about this API, see the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-monitoring-archive-api/minor/1.0/pages/home/[Anypoint Monitoring Archive API] page in Anypoint Exchange.
```

File: modules/ROOT/pages/api-analytics-dashboard.adoc [hunk 8]
```diff
@@ -26,9 +26,15 @@ image::api-analytics-detail-panel.png[]
 +
 . If you want to view API analytics in API Manager, click the *View API in API Manager* link.
 
+[[enhanced-api-metrics]]
 == Enhanced Metrics View for API Analytics Dashboards
 
-API analytics dashboards include various pages to view different types of charts:
+API analytics dashboards include various pages to view different types of charts.
+
+Titanium users who use the enhanced API analytics dashboards can also access the data directly via the Anypoint Monitoring Archive API. With this API, you can navigate and discover your API metrics data using a directory hierarchy.
```
[the remark this answers is not in the document]
Comment (kbostrom) [reply modules/ROOT/pages/api-analytics-dashboard.adoc:34]: Consider adding a paragraph break here,

File: modules/ROOT/pages/api-analytics-dashboard.adoc [hunk 9]
```diff
@@ -26,9 +26,15 @@ image::api-analytics-detail-panel.png[]
...
+The API allows you to download files that contain https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-monitoring-archive-api/minor/1.0/pages/Metric%20Data/[JSON objects] that represent one or more metric events. You can also write an application to navigate the data collected by the API and integrate it into your own system. For more information about this API, see the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-monitoring-archive-api/minor/1.0/pages/home/[Anypoint Monitoring Archive API] page in Anypoint Exchange.
```
Comment (kbostrom) [inline modules/ROOT/pages/api-analytics-dashboard.adoc:35]:
```suggestion
The Anypoint Monitoring Archive API enables you to download files that contain https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-monitoring-archive-api/minor/1.0/pages/Metric%20Data/[JSON objects] that represent one or more metric events. You can also write an application to navigate the data collected by the API and integrate it into your own system. +
For more information about this API, see the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-monitoring-archive-api/minor/1.0/pages/home/[Anypoint Monitoring Archive API] page in Anypoint Exchange.
```

File: modules/ROOT/pages/api-analytics-dashboard.adoc [hunk 10]
```diff
@@ -26,9 +26,16 @@ image::api-analytics-detail-panel.png[]
 +
 . If you want to view API analytics in API Manager, click the *View API in API Manager* link.
 
+[[enhanced-api-metrics]]
 == Enhanced Metrics View for API Analytics Dashboards
 
-API analytics dashboards include various pages to view different types of charts:
+API analytics dashboards provide you with various pages from which you can view different types of charts.
+
+If you are a Titanium user who uses the enhanced API analytics dashboards, can also access the data directly via the Anypoint Monitoring Archive API. Using this API, you can navigate and discover your API metrics data using a directory hierarchy.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/api-analytics-dashboard.adoc:34]:
```suggestion
If you are a Titanium user who uses the enhanced API analytics dashboards, you can also access the data directly via the Anypoint Monitoring Archive API. Using this API, you can navigate and discover your API metrics data using a directory hierarchy.
```

File: modules/ROOT/pages/api-analytics-dashboard.adoc [hunk 11]
```diff
@@ -157,6 +160,14 @@ Requests by Client IP:: The sum of requests in the period, grouped by client IP
 * If you select a time range within the last day, data is aggregated in ten minute increments.
 * If you select a time range that is larger than one day, data is aggregated in one hour increments.
 
+== Anypoint Monitoring Archive API for Enhanced API Analytics
+
+If you are a Titanium user who uses the enhanced API analytics dashboards, you can also access the data directly via the Anypoint Monitoring Archive API. Using this API, you can navigate and discover your API metrics data using a directory hierarchy.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/api-analytics-dashboard.adoc:165]:
```suggestion
Titanium users can also access API analytics data directly via the Anypoint Monitoring Archive API. Using this API, you can navigate and discover your API metrics data using a directory hierarchy.
```

**Pull request merged**